xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/contentview.vm

```diff
@@ -23,6 +23,9 @@ $services.progress.endStep()
 $services.progress.startStep('Display the title and content')
 <div class="xcontent">
   #template('contentheader.vm')
+  #foreach ($uix in $services.uix.getExtensions('org.xwiki.platform.template.content.header.after'))
+    $services.rendering.render($uix.execute(), 'xhtml/1.0')
+  #end
```

Comment:
isn't that what you did already in the previous PR?

Reply:
what's the relationship with "XWIKI-15112 - Add an XWiki.AccessDenied customization option"?

Reply:
Indeed, I wrongly pushed a commit that related to another PR, sorry, I'll remove the change in the next commit.